task/dao.py

```diff
@@ -89,8 +94,14 @@ def create(workspace_id: str, name: str, description: str, tags: list = None, du
 def get_by_id(id):
     """
     Get the task by id
+    If the task does not exist: -> return None
+    Else return the task object
     """
-    return get_document_by_id(Task, id)
+    try:
+        task = get_document_by_id(Task, id)
+    except ValidationError:
```

Comment:
can this actually happen? I believe get_document_by_id should do this same catch/check